### PR TITLE
fix: remove flash on creating new thread and switching threads

### PR DESCRIPTION
**Pull request by ryanhopperlowe**
Signed-off-by: Ryan Hopper-Lowe <ryan@acorn.io>

https://github.com/user-attachments/assets/f7a885e5-2e46-420a-925a-f7b42a5fbaaf

### CODE REVIEW

File: ui/admin/app/components/chat/ChatContext.tsx [hunk 1]
```diff
@@ -239,6 +238,8 @@ export function ChatProvider({
         };
     }, [generatingMessage, invokeAgent.isLoading, invokeAgent.data]);
 
+    useLogEffect(messages, insertedMessages);
```
Comment (tylerslaton) [inline ui/admin/app/components/chat/ChatContext.tsx]: Needed?

File: ui/admin/app/lib/service/api/threadsService.ts [hunk 2]
```diff
@@ -44,6 +44,9 @@ getThreadsByAgent.key = (agentId?: Nullish<string>) => {
 };
 
 const getThreadEvents = async (threadId: string) => {
+    // todo(ryan) remove debug
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```
Comment (tylerslaton) [inline ui/admin/app/lib/service/api/threadsService.ts]: Needed?

File: ui/admin/app/lib/service/api/invokeService.ts [hunk 3]
```diff
@@ -10,6 +10,8 @@ async function invokeWithStream({
     prompt?: string | null;
     thread?: string | null;
 }) {
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```
Comment (tylerslaton) [inline ui/admin/app/lib/service/api/invokeService.ts]: Needed?

File: ui/admin/app/components/chat/ChatContext.tsx (hunk 1 above)
Comment (ryanhopperlowe) [reply ui/admin/app/components/chat/ChatContext.tsx]: nope!

**Pull request merged**